MinimalHamiltonianPath/README.md

```diff
@@ -58,6 +58,7 @@ vi task.json
         "wasm_name": "mhp.wasm",
         "input_dir": "/home/lukaszglen/wasm_test_5/t2/in",
         "output_dir": "/home/lukaszglen/wasm_test_5/t2/out",
+        "__comment":"there are three subtasks, but there will be six jobs for the verification purpose (VbR)",
```

Comment:
It might be better if the README agreed with the task.json provided (the latter says two tasks)

Reply:
But README part is about adding more tasks. That was before my comments.